[LINUX]tgbdual-libretro.so not linked to libstdc++

Hello,
Curent tgbdual-libretro.so doesn't load because of missing symbols:
retroarch log:
`[ERROR] Échec de l'ouverture du cœur Libretro: "/usr/libexec/libretro/tgbdual_libretro.so"
[ERROR] Error(s): /usr/libexec/libretro/tgbdual_libretro.so: undefined symbol: _ZTVN10__cxxabiv117__class_type_infoE`
ldd -r output:
`ldd -r tgbdual_libretro.so 
        linux-vdso.so.1 (0x00007ffd977c0000)
        libgcc_s.so.1 => /lib64/libgcc_s.so.1 (0x00007f799c9c1000)
        libc.so.6 => /lib64/libc.so.6 (0x00007f799c5ec000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f799ce18000)
undefined symbol: _ZTVN10__cxxabiv117__class_type_infoE (./tgbdual_libretro.so)
undefined symbol: _ZTVN10__cxxabiv120__si_class_type_infoE      (./tgbdual_libretro.so)
undefined symbol: __gxx_personality_v0  (./tgbdual_libretro.so)
undefined symbol: _ZdlPv        (./tgbdual_libretro.so)
undefined symbol: __cxa_guard_release   (./tgbdual_libretro.so)
undefined symbol: _ZNSt8__detail15_List_node_base9_M_unhookEv   (./tgbdual_libretro.so)
undefined symbol: _Znwm (./tgbdual_libretro.so)
undefined symbol: _ZNSt8__detail15_List_node_base7_M_hookEPS0_  (./tgbdual_libretro.so)
undefined symbol: __cxa_guard_acquire   (./tgbdual_libretro.so)`
That's because the lib isn't linked to libstdc++, I fixed it by adding `-lstdc++` at line 51 of Makefile.
before: `SHARED := -shared -Wl,--version-script=libretro/link.T`
after: `SHARED := -lstdc++ -shared -Wl,--version-script=libretro/link.T`
Maybe it's not the right place but it works.
I hope it will be fixed in repo soon so online updater will have usable version.
Thanks.

--- a/Makefile
+++ b/Makefile
@@ -361,7 +361,7 @@ else
 	LD = link.exe
 endif
 else
-	LD = $(CC)
+	LD = $(CXX)
 endif
 
 INCFLAGS += $(INCFLAGS_PLATFORM)

